index.html

```diff
@@ -0,0 +1,183 @@
+                          margin-right: 18px;}
+            #notification{
+                float:left;
+                position: relative;
+                top:9px;
+                margin-right: 28px;
+            }
+            #acompte_sigle{
+                height: 26px;
+                position: relative;
+                top : 6px;
+            }
+            #suggestion_comments{
+                margin-left : 50px;
+                margin-right: 30px;
+
+            }
+            #comments{
+                float: left;
+                width : 750px;
+                margin-right: 30px;
+            }
+            #comments h2 {
+                font-weight: normal;
+                font-size: 18px;
+            }
+            #visibility{
+                background-color: #ECECEC;
+                width: 120px;
+                text-align: center;
+                color: #606060;
+                font-size: 12px;
+                font-weight: bold ;
+                padding-top: 2px;
+                padding-bottom: 2px;
+            }
+            #joint_image{
+                position: relative;
+                top : 3px;
+            }
+            #views {
+                color: #606060;
+                font-size: 15px;
+            }
+            sup{font-weight: bold;
+                font-size: 18px;}
+            hr{
+                border-color: #f9f9f9;
+                height: 1px;
+            }
+            #yot{float: left;margin-right: 10px;}
+            #subscribed{
+                float: left;
+                position: relative;
+                top : -8px;
+            }
+            span {
+                color : #646464;
+                font-weight: normal;
+                font-size: 12px;
+            }
+            #registration {
+                text-transform: uppercase;
+                color : white;
+                background-color: #CC3532;
+                width : 120px;
+                float:right;
+                text-align: center;
+                vertical-align: middle;
+                padding-top: 7px;
+                padding-bottom: 7px;
+            }
+            #suggestions {
+                float:left;
+                width:350px;
+            }
+            .all{
+                background-color: black;
+                color: white;
+                width: 60px;
+                text-align: center;
+                border-radius: 16px;
+                height : 25px;
+                padding-top:5px;
+                margin-right: 20px;
+            }
+            .seen{
+                background-color:#D6D6D6;
+                color: black;
+                width: 100px;
+                text-align: center;
+                border-radius: 16px;
+                height : 25px;
+                padding-top:5px;
+            }
+            #selection{
+                display: flex;
+                direction: row;
+                
+            }
+        </style>
```

Comment:
Kindly create an external file for css styling. It would be also required by linter.